src/core/ledger/ledger.js

```diff
@@ -33,7 +33,20 @@ import {
 import * as G from "./grain";
 import {JsonLog} from "../../util/jsonLog";
 import * as C from "../../util/combo";
-import {type Allocation, type AllocationId} from "./grainAllocation";
+import {
+  type Allocation,
+  type AllocationId,
+  type GrainReceipt,
+} from "./grainAllocation";
+
+/**
+ * Timestamped record of a grain payment made to an Identity from a specific Allocation.
+ */
+type AllocationReceipt = {|
+  allocationId: AllocationId,
+  grainReceipt: GrainReceipt,
+  timestampMs: TimestampMs,
```

Comment:
Consider changing this from `timestampMs` to `credTimestampMs` so that later we can add `ledgerTimestampMs` without undue confusion.

Reply:
accepted